computation/src/main/java/com/powsybl/computation/ComputationConfigBuilder.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.computation;
+
+import com.google.common.base.Preconditions;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ */
+public class ComputationConfigBuilder {
+
+    private final Map<String, Integer> timeoutMap = new HashMap<>();
```

Comment:
Timeout is redondant with one defined in the command. I think you should remove it.

Reply:
Maybe we should deprecated the timeout of the command. The estimated time of same computation could be different based on the platform.

Reply:
And more, the timeout could be configued with qos in a file and loaded into ComputationOptions.